# Message for setting the angular velocites of the motors
# A positive pwm means forward and a negative pwm means reverse
float64 left_angular_velocity
float64 right_angular_velocity